include/onnxruntime/core/graph/schema_registry.h

```diff
@@ -120,7 +130,7 @@ class SchemaRegistryManager : public onnxruntime::IOnnxRuntimeOpSchemaCollection
  public:
   /**
   Register a new schema registry instance.
-  @remarks The schema registry priority is the reverse of registration order. i.e. the last registry added will be
+  @remarks The schema registry priority is the order of registration order. i.e. the first registry added will be
```

Comment:
The original comment doesn't conform to the code.

Reply:
The search order doesn't matter if the registries don't have overlap on (domain,version_range). And that should be the most of cases. Because each registry should represent  a FULL opset.